src/index.js

```diff
@@ -107,90 +137,23 @@ async function authenticate(username, password) {
   }
 }
 
-function fetchFiles(year) {
-  log('info', `Fetching files paie for year ${year}`)
-  return request.get(`${baseUrl}/prive/remunerationpaie/v1?annee=${year}`, {
-    json: true
+async function getDocs() {
+  await request({
+    uri: `${baseUrl}/prive/initialiserhabilitation/v1`,
+    method: 'POST',
+    body: {},
```

Comment:
Is this body attribute really needed ?

Reply:
yes, i can't tell you why but if removed this, the site throw a "technical error".